### PR TITLE
Feature/widget mini calendar and week tweaks

### PR DESCRIPTION
**Pull request by Camwyn**
This adds some logic to insert the date separators for each day shown.
Matters most for widget initial load.

https://d.pr/i/dCQIjs

Make grid days square on small screens.
https://d.pr/v/R2Aeuj

### CODE REVIEW

File: src/views/v2/month/mobile-events/mobile-day.php [hunk 1]
```diff
@@ -49,13 +51,19 @@
...
+
+		<?php if ( empty( $running_date ) || $running_date !== $event_date ) : ?>
+			<?php $this->template( 'month/mobile-events/mobile-day/day-marker', [ 'day_date' => $event_date ] ); ?>
+		<?php endif; ?>
+
 		<?php $this->setup_postdata( $event ); ?>
 
 		<?php $this->template( 'month/mobile-events/mobile-day/mobile-event', [ 'event' => $event ] ); ?>
 
+		<?php $running_date = $event_date; ?>
```
Comment (bordoni) [inline src/views/v2/month/mobile-events/mobile-day.php:66]: Would love to see these changes be added via some sort of extra template included into before `month/mobile-events/mobile-day/mobile-event`.

**Review by bordoni — CHANGES_REQUESTED** [review]

File: src/views/v2/month/mobile-events/mobile-day.php (hunk 1 above)
Comment (Camwyn) [reply src/views/v2/month/mobile-events/mobile-day.php:66]: Not sure I agree - unless we specifically don't want this to happen on "normal" month view?

Comment (Camwyn) [reply src/views/v2/month/mobile-events/mobile-day.php:66]: There is one caveat around multiday events.
current: https://d.pr/i/ahWMmH
this change: https://d.pr/i/xZRGME
...as the event date is always the start day.

Comment (paulmskim) [reply src/views/v2/month/mobile-events/mobile-day.php:66]: see `src/views/v2/list/month-separator.php`, we use `Tribe\Events\Views\V2\Utils\Separators` to do this sort of logic.

Comment (Camwyn) [reply src/views/v2/month/mobile-events/mobile-day.php]: Done. See https://github.com/the-events-calendar/the-events-calendar/pull/3485/files#diff-f745177a36763b9a049d9d3973bb84e45bc46ffb6849a399c231da9aaf6b7703

File: src/Tribe/Views/V2/Widgets/Widget_List.php [hunk 2]
```diff
@@ -104,6 +104,40 @@ protected function setup_default_arguments() {
...
+	 * Add this widget's css group to the VE list of widget groups to load icon styles for.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string> $widgets The list of widgets
+	 *
+	 * @return array<string> The modified list of widgets.
+	 */
+	public function add_self_to_virtual_widget_groups( $groups ) {
+		$groups[] = self::get_css_group();
```
Comment (bordoni) [inline src/Tribe/Views/V2/Widgets/Widget_List.php:136]:
```suggestion
		$groups[] = static::get_css_group();
```

**Review by bordoni — CHANGES_REQUESTED** [review]
One small change.

**Pull request merged**